src/sage/schemes/affine/affine_space.py

```diff
@@ -46,12 +46,17 @@ def is_AffineSpace(x) -> bool:
 
         sage: from sage.schemes.affine.affine_space import is_AffineSpace
         sage: is_AffineSpace(AffineSpace(5, names='x'))
+        doctest:warning...
+        DeprecationWarning: the function is_AffineSpace is deprecated; use 'isinstance(..., AffineSpace_generic)' instead
+        See https://github.com/sagemath/sage/issues/38022 for details.
         True
         sage: is_AffineSpace(AffineSpace(5, GF(9, 'alpha'), names='x'))                 # needs sage.rings.finite_rings
         True
         sage: is_AffineSpace(Spec(ZZ))
         False
     """
+    from sage.misc.superseded import deprecation
+    deprecation(38022, "the function is_AffineSpace is deprecated; use 'isinstance(..., AffineSpace_generic)' instead")
     return isinstance(x, AffineSpace_generic)
```

Comment:
According to the developer guide 

https://doc.sagemath.org/html/en/developer/coding_in_python.html#experimental-unstable-code

deprecation message begins with a capital letter and ends in a period.

Reply:
Thanks for the pointer; I've done this wrong in a bunch of other recent PRs, I think.

Fixed in ceebb70482a

Reply:
Perhaps I also have missed it before. This time I could see the style distinguished from that of the surrounding text.

Thanks.